kubernetes/client.go

```diff
@@ -130,7 +130,12 @@ func getConfigForLocalCluster() (*rest.Config, error) {
 	remoteSecretPath := kialiconfig.Get().Deployment.RemoteSecretPath
 	if remoteSecret, readErr := GetRemoteSecret(remoteSecretPath); readErr == nil {
 		log.Debugf("Using remote secret for local cluster config found at: [%s]. Kiali must be running outside the kube cluster.", remoteSecretPath)
-		return clientcmd.NewDefaultClientConfig(*remoteSecret, nil).ClientConfig()
+		cc, err := clientcmd.NewDefaultClientConfig(*remoteSecret, nil).ClientConfig()
+		if err != nil {
+			return cc, err
+		}
+		cc.BearerTokenFile = remoteSecretPath
```

Comment:
Same comment here about not overriding what the user provides in the remote kube config. We should be able to verify that if you set `SecretFile` in your kubeconfig that `client-go` will in turn set `BearerTokenFile`.

Reply:
let me check that. I was assuming it wasn't going to set that under the covers. But I'll check. This could be why I am getting errors in CI.

Reply:
I removed this